components/form/DatePickerInput.tsx

```diff
@@ -9,8 +9,17 @@ const DatePickerInput = (props: Props) => {
   const [toDate, setToDate] = useState("");
 
   useEffect(() => {
-    const startDate = fromDate ? `${fromDate}T00:00` : "";
-    const endDate = toDate ? `${toDate}T23:59` : "";
+    const startDate = fromDate ? `${fromDate}T02:00` : "";
+
+    let endDate = toDate ? `${toDate}T00:00` : "";
+
+    if (endDate) {
+      const endDateObj = new Date(endDate);
+      endDateObj.setDate(endDateObj.getDate() + 2);
+      endDateObj.setHours(endDateObj.getHours() - 22);
+      endDate = endDateObj.toISOString();
+    }
```

Comment:
trenger vi alt dette? kan vi ikke bare bruke `endDate` og sette dagen til +1?

noe sånn:
```suggestion
    constendDate = toDate ? `${toDate}T02:00` : "";

    if (endDate) {
      endDate.setDate(endDateObj.getDate() + 1);
    }
```

Også bare `toISOString()` på start og end date før de returneres